cli/pkg/state/vmclarity.go

```diff
@@ -137,6 +137,26 @@ func (v *VMClarityState) MarkDone(ctx context.Context, errors []error) error {
 	return nil
 }
 
+func (v VMClarityState) IsAborted(ctx context.Context) (bool, error) {
+	scanResult, err := v.client.GetScanResult(ctx, v.scanResultID, models.GetScanResultsScanResultIDParams{
+		Select: utils.PointerTo("id,status"),
+	})
+	if err != nil {
+		return false, fmt.Errorf("failed to get scan result: %w", err)
+	}
+
+	state, ok := scanResult.GetGeneralState()
+	if !ok {
+		return false, errors.New("failed to get general state of scan result")
```

Comment:
please be consistent with `fmt.Errorf`/`errors.New`

Reply:
@FrimIdan You mean starting error messages with capital letter?

Reply:
I mean the use of `errors.New` and `fmt.Errorf`